workbench-core/environments/src/interfaces/createEnvironmentTypeConfigRequest.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const CreateEnvironmentTypeConfigRequestParser = z
+  .object({
+    envTypeId: z.string(),
+    params: z
```

Comment:
Do we want to keep body as `params` or become its own properties?

Reply:
If the object gets reused (or we expect it to be reused) maybe pull it out? But if its just a one time thing then leave as is?

Reply:
moved to top level in last commit